public/js/showoff.js

```diff
@@ -436,7 +441,7 @@ function determineIncremental()
 		incrCode = true
 	}
 	incrElem.each(function(s, elem) {
-		$(elem).css('visibility', 'hidden');
+		$(elem).addClass('incremental hidden');
```

Comment:
Not sure if this will be an issue: 
  Adding 'incremental hidden' here, but only removing 'hidden' in showIncremental.  Could this potentially lead to incremental being added multiple times to the same element?